### PR TITLE
Use pbr module and semantic versioning

### PR DESCRIPTION
**Pull request by tareqalayan**

### CODE REVIEW

**Comment by tareqalayan** [comment]
@lukas-bednar

File: setup.cfg [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    plugin
+[files]
+packages =
+    pytest_marker_bugzilla
+[bdist_rpm]
+requires =
+    pytest >= '2.2.4'
+    python-bugzilla >= '0.6.2'
+    pbr >= '0.8.2'
+    six
```
Comment (lukas-bednar) [inline setup.cfg]: you are missing entry_points .... that is why tests failed, because plugin is not visible for pytest ...

File: setup.cfg [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+author-email = elsammons@gmail.com
+summary = py.test bugzilla integration plugin, using markers
+description-file = README.md
+home-page = https://github.com/eanxgeek/pytest_marker_bugzilla.git
+license = GPL2
+zip_safe=False
+classifier =
+    Development Status :: 3 - Alpha
+    Topic :: Utilities
+    License :: OSI Approved :: GNU General Public License 2 (GPL2),
```
Comment (lukas-bednar) [inline setup.cfg]: please use exact string
`License :: OSI Approved :: GNU General Public License v2 (GPLv2)`

File: setup.cfg (hunk 1 above)
Comment (lukas-bednar) [inline setup.cfg]: https://travis-ci.org/rhevm-qe-automation/pytest-marker-bugzilla/jobs/127988689

**Comment by lukas-bednar** [comment]
@eanxgeek could you please enable gating job on travis CI? It is supper  easy, you have all ready, just go to that site, log in with github account, and press enable button for this repository ...

It will show results directly in PR, and you can see whether change breaks something or not ...

For example this change breaks your plugin, because there are missing 'entry_points' in setup.cfg, so pytest can not discover plugin. And you have no feedback about it right now ...

https://travis-ci.org/

File: setup.cfg [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+author = Eric Sammons
+author-email = elsammons@gmail.com
+summary = py.test bugzilla integration plugin, using markers
+description-file = README.md
+home-page = https://github.com/eanxgeek/pytest_marker_bugzilla.git
+license = GPL2
+zip_safe=False
+classifier =
+    Development Status :: 3 - Alpha
+    Topic :: Utilities
```
Comment (lukas-bednar) [inline setup.cfg]: double same topic, next occurrence on line 19

File: setup.cfg [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+description-file = README.md
+home-page = https://github.com/eanxgeek/pytest_marker_bugzilla.git
+license = GPLv2
+zip_safe=False
+entry_points={
+        'pytest11': ['pytest_marker_bugzilla = pytest_marker_bugzilla'],
+    }
+classifier =
+    Development Status :: 3 - Alpha
+    License :: OSI Approved :: GNU General Public License 2 (GPLv2),
```
Comment (lukas-bednar) [inline setup.cfg]: still wrong, you can just copy it ...

`License :: OSI Approved :: GNU General Public License v2 (GPLv2)`

File: setup.cfg [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+    Operating System :: POSIX,
```
Comment (lukas-bednar) [inline setup.cfg]: all trailing commas in this section are wrong ... must be deleted

File: setup.cfg [hunk 6]
```diff
@@ -0,0 +1,38 @@
+[metadata]
+name = pytest-marker-bugzilla
+author = Eric Sammons
+author-email = elsammons@gmail.com
+summary = py.test bugzilla integration plugin, using markers
+description-file = README.md
+home-page = https://github.com/eanxgeek/pytest_marker_bugzilla.git
+license = GPLv2
+zip_safe=False
+entry_points={
```
Comment (lukas-bednar) [inline setup.cfg]: this is not correct for setup.cfg ..

you can find example in  doc, just take a look

http://docs.openstack.org/developer/pbr/#setup-cfg

**Comment by eanxgeek** [comment]
I have turned on travis-ci for this project.

**Comment by eanxgeek** [comment]
I'm closing this pull request and not incorporating.  Please see comments and resubmit so that CI can validate.

File: requirements.txt [hunk 7]
```diff
@@ -0,0 +1,3 @@
+pytest >= 2.2.4
+python-bugzilla >=0.6.2
+pbr >= 0.8.2
```
Comment (lukas-bednar) [inline requirements.txt]: missing 'six'

File: setup.cfg [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+    bugzilla
+    plugin
+[files]
+packages =
+    pytest_marker_bugzilla
+[bdist_rpm]
+requires =
+    pytest >= '2.2.4'
+    python-bugzilla >= '0.6.2'
+    pbr >= '0.8.2'
```
Comment (lukas-bednar) [inline setup.cfg]: python-pbr

File: setup.cfg (hunk 1 above)
Comment (lukas-bednar) [inline setup.cfg]: python-six

**Pull request closed without merging**